.github/workflows/ci.yml

```diff
@@ -273,3 +273,9 @@ jobs:
           node-version: 16.x
       - run: yarn install --frozen-lockfile
       - run: npx semantic-release
+      - id: sequelize
+        uses: sdepold/github-action-get-latest-release@master
```

Comment:
Let's update it to https://github.com/pozetroninc/github-action-get-latest-release once your PR is merged

Reply:
If you don't mind, I would rather not wait since GitHub Actions are essentially just repositories that are cloned during runtime. There is no real point in waiting for the PR to be merged

Reply:
Agree, merge this now but in a future PR put it back to the original when your PR is merged by pozetroninc